test/api/utils.js

```diff
@@ -15,6 +15,55 @@
 import utils from '../../src/api/utils';
 
 describe('api utils module', () => {
+	const port = 7000;
+	const localNode = 'localhost';
+	const externalNode = 'external';
+	const sslNode = 'sslPeer';
+	const externalTestnetNode = 'testnet';
+	const GET = 'GET';
+	const POST = 'POST';
+	const defaultMethod = POST;
+	const defaultEndpoint = 'transactions';
+	const defaultPeers = [localNode, externalNode];
+	const defaultSSLPeers = [localNode, externalNode, sslNode];
+	const defaultTestnetPeers = [localNode, externalTestnetNode];
+
+	let LSK;
+	let sendRequestResult;
+	let sendRequestStub;
+
+	beforeEach(() => {
+		LSK = {
+			randomPeer: false,
+			currentPeer: localNode,
+			defaultPeers: [].concat(defaultPeers),
+			defaultSSLPeers: [].concat(defaultSSLPeers),
+			defaultTestnetPeers: [].concat(defaultTestnetPeers),
+			bannedPeers: [],
+			port,
+			options: {
+				node: localNode,
+			},
+			nethash: {
+				foo: 'bar',
+			},
+
+			parseOfflineRequests: () => ({
+				requestMethod: GET,
+			}),
+			setTestnet: () => {},
+			setNode: () => {},
+			sendRequest: () => {},
+		};
```

Comment:
This `LSK` mock can be cut down a lot. Try to get rid of everything we don't need for testing utils.